pyproject.toml

```
@@ -1,15 +1,19 @@
-[build-system]
-requires = [
-    "setuptools>=65",
-    "wheel>=0.38",
-]
-build-backend = "setuptools.build_meta"
-
-[project]
+[tool.poetry]
 name = "pyflexplot"
-version = "1.1.1"
+# Keep only Major and Minor version in this configuration,
+# the patch version is calculated when releasing the library
+version = "2.0"
 description = "PyFlexPlot - Visualize and post-process FLEXPART dispersion simulation results stored in NetCDF format"
+authors = ["Pirmin Kaufmann <pirmin.kaufmann@meteoswiss.ch>", "Stefan Ruedisuehli <stefan.ruedisuehli@env.ethz.ch>"]
```

Stefan has written the vast majority of this code and should be the first author
```suggestion
authors = ["Stefan Ruedisuehli <stefan.ruedisuehli@env.ethz.ch>", "Pirmin Kaufmann <pirmin.kaufmann@meteoswiss.ch>"]
```